baselines/Common Controls Minimum Viable Secure Configuration Baseline v0.1.md

```diff
@@ -1218,16 +1093,6 @@ The following critical logs SHALL be sent at a minimum.
   - [T1562: Impair Defenses](https://attack.mitre.org/techniques/T1562/)
     - [T1562:008: Impair Defenses: Disable Cloud Logs](https://attack.mitre.org/techniques/T1562/008/)
 
-#### GWS.COMMONCONTROLS.16.2v0.1
-Audit logs SHALL be maintained for at least 6 months in active storage and an additional 18 months in cold storage, as dictated by OMB M-21-31. The logs SHALL be sent to the agency's Security Operations Center (SOC) for monitoring.
-
-- Rationale
-  - OMB M-21-31, Improving the Federal Government's Investigative and Remediation Capabilities Related to Cybersecurity Incidents, provides guidance on log retention for federal agencies. The memorandum defines three maturity levels for log retention, with each level requiring different minimum retention periods.
-- Last Modified: July 10, 2023
-
-- MITRE ATT&CK TTP Mapping
-  - [T1562: Impair Defenses](https://attack.mitre.org/techniques/T1562/)
-    - [T1562:008: Impair Defenses: Disable Cloud Logs](https://attack.mitre.org/techniques/T1562/008/)
```

Comment:
I'd actually argue against deleting this one. My original feedback with regard to this policy is that we might want to rework it so that a single policy doesn't have two separate SHALL statements. Removing just the "The logs SHALL be sent to..." part would accomplish that.

Reply:
@adhilto This policy has already deleted and deferred to hardening guide.